model/postgres/postgres_timeline.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require_relative "../../model"
+
+class PostgresTimeline < Sequel::Model
+  one_to_one :strand, key: :id
+  one_to_one :parent, key: :parent_id, class: self
+
+  include ResourceMethods
+  include SemaphoreMethods
+
+  semaphore :destroy
+
+  plugin :column_encryption do |enc|
+    enc.column :access_key
+    enc.column :secret_key
+  end
+
+  def bucket_name
+    ubid
+  end
+
+  def blob_storage_client
+    @blob_storage_client ||= MinioClient.new(
+      endpoint: Config.postgres_service_blob_storage_endpoint,
+      access_key: Config.postgres_service_blob_storage_access_key,
+      secret_key: Config.postgres_service_blob_storage_secret_key
```

Comment:
The goal was generating new `access_key` and `secret_key` in the `generate_walg_config` using MinIO client, which doesn't have this functionality now. I didn't do it in `assemble`, because we need to rotate them anyway, so `generate_walg_config` is the appropriate place to set those secrets.

However, for now, I agree it fits better to set them in `assemble`. I changed it. We can revisit here once we have better MinIO client.